nix/docker/runner

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+set -exo pipefail
+
+filer
+
+export OMNIA_CONFIG=$HOME/omnia.conf
+confer OMNIA > $OMNIA_CONFIG
+
+mkdir -p $HOME/.ssb
+confer SSB > $HOME/.ssb/config
+
+mkdir -p $HOME/logs
+
+ssb-server start 2>$HOME/logs/ssb-server.err >$HOME/logs/ssb-server.out &
+sleep 5
+
+test -z "$SSB_INVITE" || {
+  ssb-server invite.accept "$SSB_INVITE"
+  sleep 1
+}
+
```

Comment:
Running into this here - https://github.com/makerdao/oracles-v2/pull/55/files#diff-79738685a656fe6b25061bb14181442210b599f746faeaba408a2401de45038aR61

SSB server becomes unresponsive after accepting the invite and Omnia can't start:
```
2020-10-30T06:51:18 my key ID: J/y5JDBNy0woXv/EwAOrUnn9em4SljVf9sElnAaK62s=.ed25519
2020-10-30T06:51:38 ==> /home/omnia/logs/omnia.err <==
2020-10-30T06:51:38 [10/30/20 06:51:38] [E] Could not get scuttlebot feed id, make sure scuttlebot server is running
2020-10-30T06:51:46 ==> /home/omnia/logs/ssb-server.err <==
2020-10-30T06:51:46 server error, from net:::ffff:127.0.0.1:38938~shs:
2020-10-30T06:51:46 Error: stream ended with:64 but wanted:112
2020-10-30T06:51:46     at drain (/nix/store/1fs5x4j38l702azl2sw0i4m0kavmf1m9-node_ssb-server-15.2.0/lib/node_modules/ssb-server/node_modules/pull-reader/index.js:43:26)
2020-10-30T06:51:46     at /nix/store/1fs5x4j38l702azl2sw0i4m0kavmf1m9-node_ssb-server-15.2.0/lib/node_modules/ssb-server/node_modules/pull-reader/index.js:63:18
2020-10-30T06:51:46     at /nix/store/1fs5x4j38l702azl2sw0i4m0kavmf1m9-node_ssb-server-15.2.0/lib/node_modules/ssb-server/node_modules/pull-reader/index.js:20:7
2020-10-30T06:51:46     at drain (/nix/store/1fs5x4j38l702azl2sw0i4m0kavmf1m9-node_ssb-server-15.2.0/lib/node_modules/ssb-server/node_modules/stream-to-pull-stream/index.js:126:18)
2020-10-30T06:51:46     at Socket.<anonymous> (/nix/store/1fs5x4j38l702azl2sw0i4m0kavmf1m9-node_ssb-server-15.2.0/lib/node_modules/ssb-server/node_modules/stream-to-pull-stream/index.js:143:5)
2020-10-30T06:51:46     at Socket.emit (events.js:203:15)
2020-10-30T06:51:46     at endReadableNT (_stream_readable.js:1145:12)
2020-10-30T06:51:46     at process._tickCallback (internal/process/next_tick.js:63:19)
```

Reply:
Would you consider putting that `until` look here?

Reply:
Yes, will do that, thanks!

Reply:
This is now working 👍